clients/flash/air-client/src/org/bigbluebutton/air/chat/views/TopToolbarMediatorChat.as

```diff
@@ -23,10 +20,6 @@ package org.bigbluebutton.air.chat.views {
 			}
 		}
 		
-		override protected function leftButtonClickHandler(e:MouseEvent):void {
-			uiSession.popPage();

```

Comment:
Popping the page is the correct thing to do. If you're on the chat view the stack is Main > Participants > Chat. Popping the last one gets you back to where you were rather than adding another layer to the stack.